protocol/avdl/stellar1/local.avdl

```diff
@@ -219,6 +219,10 @@ protocol local {
     // Example: "This stellar address is incorrect"
     string toErrMsg;
 
+    // Optional. `arg.to` may be an account ID that resolves to a user.
+    // If this is returned, caller should switch to (to=res.toUsername,toIsAccountID=false).
+    string toUsername;
+
```

Comment:
@cjb does this addition make sense? See the ticket for more context https://keybase.atlassian.net/browse/CORE-8286

Reply:
Yes!